app/models/consent_form.rb

```diff
@@ -23,34 +23,57 @@
 #
 
 class ConsentForm < ApplicationRecord
-  attr_accessor :is_this_their_school
+  cattr_accessor :form_steps do
+    %i[name date_of_birth school]
+  end
```

Comment:
This is the new source of truth for consent form steps; I like that it's in the same place as the validations.